internal/builder/packages.go

```diff
@@ -233,7 +240,53 @@ func signZippedPackage(options BuildOptions, zippedPackagePath string) error {
 	return nil
 }
 
+func copyLicenseTextFile(licensePath string) error {
+	_, err := os.Stat(licensePath)
+	if err == nil {
+		// License is already there, nothing to do.
+		return nil
+	}
+
+	sourceLicensePath, err := findRepositoryLicense()
+	if errors.Is(err, os.ErrNotExist) {
+		logger.Debug("No license text file is included in package.")
+		return nil
+	}
+	if err != nil {
+		return errors.Wrap(err, "failure while looking for license in repository")
+	}
+
+	logger.Debugf("License text found in %q will be included in package", sourceLicensePath)
+	err = sh.Copy(licensePath, sourceLicensePath)
+	if err != nil {
+		return errors.Wrap(err, "can't copy license from repository")
+	}
+
+	return nil
+}
+
 func createBuildDirectory(dirs ...string) (string, error) {
+	dir, err := findRepositoryRootDirectory()
+	if errors.Is(err, os.ErrNotExist) {
+		return "", errors.New("package can be only built inside of a Git repository (.git folder is used as reference point)")
+	}
```

Comment:
Does it change the implementation logic or is it just refactoring?

Reply:
Just refactoring, this is basically a copy of what there was [in the loop](https://github.com/elastic/elastic-package/pull/920/files#diff-a018efea59016591a1ef36042025a57b31c426b7f3c0c45f83b6e8af81580585L247) looking for `.git`.